### PR TITLE
[10.0][FIX] fix disambiguation in account_bank_statement_import_camt …

### PR DESCRIPTION
**Pull request by grindtildeath**
module name

Supersedes : #171 (author's github account closed)

### CODE REVIEW

**Comment by grindtildeath** [comment]
@simahawk @leemannd @pedrobaeza Can you please pass by ?

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I think we can merge this after the other approval.

**Pull request merged**
